HLTrigger/JetMET/plugins/HLTJetTimingFilter.h

```diff
@@ -0,0 +1,122 @@
+/** \class HLTJetTimingFilter
+ *
+ *  \brief  This makes selections on the timing and associated ecal cells 
+ *  produced by HLTJetTimingProducer
+ *  \author Matthew Citron
+ *
+ */
+#ifndef HLTrigger_JetMET_plugins_HLTJetTimingFilter_h
+#define HLTrigger_JetMET_plugins_HLTJetTimingFilter_h
+
+// system include files
+#include <memory>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "HLTrigger/HLTcore/interface/HLTFilter.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
```

Comment:
If you want to keep separated .h and .cc files, this can be included in the implementation file